### DISCUSSION TITLE
How to get date in different format?

### DISCUSSION THREAD

**Discussion by monoraulg** [wojtekmaj/react-daterange-picker#117] [Q&A]
Hi!
How to get the value of the dates like this: 10/12/2021 instead of the string that returns "value" which is like this:  [Tue Oct 12 2021 00:00:00 GMT-0300 (hora estándar de Argentina), Fri Oct 15 2021 23:59:59 GMT-0300 (hora estándar de Argentina)] 
The value set with the useState is always the long string.

**Comment by wojtekmaj** [comment]
`Tue Oct 12 2021 00:00:00 GMT-0300 (hora estándar de Argentina)` is not a string, it's a text representation of [`Date`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) object that is actually being returned. To format it to a string with desired format, use [`Intl.DateTimeFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat) or one of popular libraries like [date-fns](https://date-fns.org/).
